h2o-dist/index.html

```diff
@@ -677,7 +677,19 @@ <h2>Gradle-style specification for Maven artifacts</h2>
 }</br>
           </div>
             <div id="quickstart-kubernetes" style="display:none">
-                <h2>Get started with H<sub>2</sub>O on Kubernetes</h2>
+
+                <h2>Setup H<sub>2</sub>O on Kubernetes using Helm</h2>
+<a href="https://helm.sh/" target="_blank">Helm</a> can be used to deploy H2O into a kubernetes cluster. Helm requires the KUBECONFIG environment variable to be set up properly, or stating the kubeconfig destination explicitly. Please refer to Helm's documentation
```

Comment:
should text be inside a <p>, also would prefet to fix the indent and line length

Reply:
Put all of into a paragraph and that fixed the indetation :wink: Thanks.